### ISSUE TITLE
Avoid usage of prototype-rails

### ISSUE THREAD

**Issue by christianlupus** [jcppkkk/redmine_mail_reminder#97]
In the current version there is a dependency in the Gemfile of `prototype-rails`. This project is no longer maintained and not officially supported rails>=4.1(see https://github.com/rails/prototype-rails README).
I am running into issues as redmine requests rails 5.1.2 explicitly which leads to problems if I try to install redmine_mail_reminder.

Is it possible to get rid of this dependency?

**Comment by SvenRoederer** [comment]
@christianlupus have you found a solution for this?

**Comment by christianlupus** [comment]
@SvenRoederer No, unfortunately not. I fear this plugin is dead at the moment.

**Comment by SvenRoederer** [comment]
@drakosu from the repos network-graph (https://github.com/ascendro/redmine_issue_reminder/network) I found your repo as the one with the most recent commit.
Do you know o a solution for the depencency of the unmaintained prototype-rails addon?

**Comment by drakosu** [comment]
@SvenRoederer Unfortunately I am a newbie to Ruby and Rails. Hope you find a good solution.
